src/Commands/JWTGenerateSecretCommand.php

```diff
@@ -59,9 +59,9 @@ public function fire()
                     file_put_contents($path, str_replace(
                         'JWT_SECRET='.$this->laravel['config']['jwt.secret'], 'JWT_SECRET='.$key, file_get_contents($path)
                     ));
-                } else {
-                    return $this->comment('Phew... No changes were made to your secret key.');
                 }
+
+                return $this->comment('Phew... No changes were made to your secret key.');
```

Comment:
This one is necessary. If you answer "yes" then it will still output as if no changes were made